### DISCUSSION TITLE
聊天的應用程式

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1858] [General]
每個年代都會有一個比較流行用來聊天的應用程式 像是我國小的時候比較常用messenger或Line 現在則會用Discord或Instagram，大家還有在用什麼應用程式來聊天嗎？

**Comment by thomasxshen** [comment]
我都使用pronhub聊天，pronhub是一個很好的影片交流平台，在那裡，可以看到人們最赤裸裸的言論，也是世界上最和平的聊天程式，不會有無腦的謾罵和可怕的網軍攻擊，只會進行人與人之間身心靈的交流。

**Reply by yubo0oo** [reply]
支持，我也都在上面跟網友聊天，pornhub是我看過最和平的網站，沒有之一，不分國界，所有人都在上面交流

**Reply by poiu6fi** [reply]
再推一個jable，每個人共同交流自己的喜好，甚是快樂

**Comment by YANRONG0123** [comment]
老哥，我從以前到現在都是使用messenger、LINE，直到現在的Instagram、Discord聊天，經歷過各種不同非匿名與匿名的聊天程式，感覺自己的社交能量已經到達了極限，可以教學一下樓上所使用聊天平台的溝通方式嗎?

**Comment by woohsin** [comment]
高中的時候覺得LINE是老人家在用的，結果上大學才發現大家都是用LINE在傳資料，所以現在幾乎都是用LINE處理比較重要的是(也會聊天用)，如果只是平常時的話都用MESSENGER跟INSTAGRAM比較多。

**Comment by jylin34** [comment]
我都是跟家人用line，朋友聊天用ig，同學溝通學業上的事情用messenger，傳檔案用telegram(假聊天軟體，真雲端硬碟)。

**Comment by willychiuchiu** [comment]
我自己是習慣和家人、較熟的朋友用Line，然後instagram 是用來社交、回限時動態或是和比較不熟的朋友聊天，而messengers 就比較少用，大概只有買東西聯絡用。

**Comment by 110101501** [comment]
自己是IG、Messenger、LINE都會用，沒有特別偏好使用哪一款，比較會配合對方習慣的應用程式配合使用。
不過傳訊息的本質是希望對方收到自己的訊息，所以我認為目的有達成就好，使用的應用程式就相對不是這麼重要了。

**Comment by yxin214** [comment]
最喜歡用LINE！覺得LINE的版面好看又不單調，通話和視訊品質也穩，還有很多可愛的主題和貼圖。雖然IG也能更換主題和傳送貼圖，甚至不用額外花錢購買，但LINE就是讓人有種心甘情願花錢的魔力。

**Comment by wanka2030** [comment]
小時候會因為朋友都在fb上面所以messenger會是一個很棒的溝通環境，但大學比較'常用的就會變成line，除了能夠有更明確的分類，也有更加強大的功能性能夠更加符合需求，LINE的貼圖以及主題能夠使使用者有附加的物件表達自身情感。

**Comment by happy246810** [comment]
我最常使用的通訊軟體是LINE跟Discord，LINE通常都是用來跟家人聯絡跟傳遞資料，而Discord通常是用來跟同學聊天。雖然大家都說LINE是老人在用的，但我並不覺得，因為我認為LINE真的很好用、便利。

**Comment by LinChloe** [comment]
我的話 instagram 是用來社交、回限時動態或是和比較不熟的朋友聊天，很多人想加ig聊天，但我認為如果你沒加到我的FB在我的認知上就不算朋友。
如果朋友一直用ig和我聊天我會絕交。

**Comment by yyujimmy** [comment]
除了您提到的應用程式外，還有一些其他的聊天應用程式在不同的年代和群體中流行。例如：

WhatsApp：這是一個全球性的聊天應用程式，提供即時通訊、語音通話和視訊通話等功能，廣泛應用於許多國家和地區。
Telegram：這是一個安全性較高的聊天應用程式，提供端對端加密的聊天功能，同時也支援群組聊天和頻道訊息。
Snapchat：這是一個以短暫影像和照片分享為主的聊天應用程式，特色是訊息會在一段時間後自動消失。
WeChat（微信）：這是中國大陸最為流行的聊天應用程式，除了聊天功能外，還提供支付、社交、購物等多種功能。
Signal：這是一個以隱私保護為重點的聊天應用程式，提供端對端加密的訊息傳輸，廣受注目於注重隱私安全的使用者群體中。

**Comment by KH1126** [comment]
因為高中在大陸讀台商學校，所以還會用wechat。
wechat在大陸很方便，尤其是現在手機支付很普遍，出門可以帶一隻手機就好。

**Comment by William08290829** [comment]
還記得國小最常用line，然後到國中高中都是跟同學用messenger聊天，但可能到了大學發現好像身邊的人都還是用line所以又變回line了。感覺長大了大家還是比較喜歡有隱私的感覺所以line會比較大眾。

**Comment by tzurou** [comment]
我自己國中的時候都用messenger聊天，那時候拿的是安卓，臉書當時在年輕人間很盛行，但後來到了高中不知不覺就變成了IG，加上IG先推出摯友的功能，後來就開始變成用IG和朋友聊天，到現在都是這樣，LINE通常是班級、報告、家人群組，聊天對象的話題通常都是公事，也比較不會講幹話，比較親密的人才會用LINE聊天

**Comment by alexender0104** [comment]
以前我覺得line是老人家在用的通訊軟體，結果現在的我都是用line在跟朋友聊天跟互傳貼圖洗版，並且現在蠻多店家都可以使用line pay出門時只需帶一隻手機就可以做到蠻多的事。

**Comment by JustAHedgehog** [comment]
個人的使用習慣是這樣的，Line, Messenger 以及 Instagram 都是用來聊天溝通的軟體，但就是依照要聊天的對象決定使用的軟體，向有人就比較喜歡用 Messenger，也有人只能用 Instagram 才能連繫上對方。至於 Line 的部分原先是用來聊天聯繫的，但後面因為建群方便而出現一堆群組，後面默默變成公事群了（現在看到 Line 出現訊息都會緊張一下）。至於為何其他軟體沒有出現這些演化，我認為最大原因是太多出現沒有收到訊息的狀況了，像 Messenger 跟 Discord 常常收不到訊息，不然公事群真想都用Discord @@，還是有哪位大神能教導如何解決這個問題，因為他就是有時會跳出訊息有時不會

**Comment by SusuTima** [comment]
上了高中後被同學說用messenger 很老，才逐漸意識到大家已經漸漸的從FB 轉往IG使用了。不過到目前為止，還是比較習慣用messenger和LINE跟朋友家人傳訊息，LINE比較常用在處理公事上，messenger則是比較常用來跟朋友家人聊聊天。

**Comment by hbh10601** [comment]
其實我主要都還是Messenger欸，但ig和Line都有，對我來說只是一個傳訊息的媒介，不過玩遊戲的話就會都用Discord

**Comment by tshsu04224** [comment]
我個人是家人或是比較親近的朋友通常會用line聊天，而有公事或要做報告創的群組也會使用line對話，其他比較常用的就是instagram了，他的好處就是十分方便，也有很多可以回覆別人的地方，可能回個限時動態或是便利貼等等就可以聊起來，所以才會常常使用instagram聊天

**Comment by cassy0201** [comment]
以前國小國中都覺得Messenger很好用，上高中才發現Line的方便，現在不論是公事還是朋友聊天逼本上都是用Line回的訊息，而Instagram就只是拿來看大家的動態還有小聊兩句十的工具，比較少拿來溝通

**Comment by NcuJerryKao** [comment]
隨著科技的不斷發展，聊天應用程式也不斷更新與演變。除了您提到的Messenger、Line、Discord和Instagram外，還有許多其他流行的聊天應用程式，以下是一些常見的：
1. WhatsApp：一個全球流行的聊天應用程式，提供文字、語音、視訊通話等功能。
2. Telegram：一個注重隱私和安全的聊天應用程式，提供加密通訊和自毀訊息等功能。
3. Snapchat：一個以瞬間訊息和短暫訊息為特色的聊天應用程式，提供各種有趣的濾鏡和功能。
4. WeChat：一個中國流行的多功能應用程式，除了聊天功能外，還提供支付、社交、遊戲等多種功能。
5. Signal：另一個注重隱私和加密的聊天應用程式，提供安全的通訊環境。
不同的聊天應用程式有不同的特色和功能，您可以根據自己的需求和喜好選擇適合自己的聊天工具。希望這些建議對您有所幫助，讓您能夠找到適合您的聊天應用程式，與朋友和家人保持聯繫。

**Comment by OHoln** [comment]
個人認為discord的群組真的比其他軟體都方便很多，像是一些討論用的群組用discord管理起來真的會清晰很多，但遺憾的就是很多同學並不習慣使用。

**Comment by 710388** [comment]
首選line messenger 感覺任何人都可以加你 隱私性沒那麼夠

**Comment by yubo0oo** [comment]
最常用discord
line的照片會過期很討厭

**Comment by YeeDaniel** [comment]
賴好用一生推，messenger我用電腦版會有不能傳檔案的問題，蠻討厭的。IG也蠻好用的，還有超嗨留言蠻好玩的~thread也不錯~

**Comment by buoioiry** [comment]
我習慣正事一律使用LINE紀錄，因為可以用記事本或是關鍵字搜尋對話，messenger搜對話的功能我不太喜歡，ig則是用來傳垃圾影片給朋友看，或是用來跟一些沒那麼熟的人社交、看大家現實動態等，threat也會偶爾看，但上面有些對話同溫層太厚了，看幾篇就會滑回ig
